test-clients/src/main/java/com/hedera/services/bdd/suites/HapiSuite.java

```diff
@@ -39,7 +39,9 @@
 import org.apache.logging.log4j.Logger;
 
 public abstract class HapiSuite {
-
+    // The first 0 refers to the shard of the target network.
+    // The second 0 refers to the realm.
+    public static final String DEFAULT_SHARD = "0.0.";
```

Comment:
Probably rename to `DEFAULT_SHARD_REALM`

Reply:
sure no problem